biogrid/apps/api/src/app/services/Biogrid.ts

```diff
@@ -40,13 +40,14 @@ function createRandomBuildingPosition(
 export async function simulateNewBiogrid(
   body: NewBiogridOpts
 ): Promise<BiogridSimulationResults> {
-  const buildings = new Array(body.numBuildings).map((v) => {
+  const buildings: Building[] = []
+  for (let i = 0; i < body.numBuildings; i++) {
     const randomPos = createRandomBuildingPosition(
       body.townWidth,
       body.townHeight
     );
-    return new Building(10, randomPos.x, randomPos.y);
-  });
+    buildings.push(new Building(10, randomPos.x, randomPos.y));
```

Comment:
for buildings, can you use the `default_initial_energy` in `biogrid/libs/biogrid-simulator/src/lib/config/bio-constants.ts` instead of writing 10. This helps to minimize the changes which can be done when one changes the default values

Reply:
+1 :)